UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -571,16 +571,16 @@ <h4>Examples</h4>
 
 					<p>Three examples are provided for the conformance statement, one shows a statement that meets recommended accessibility requirements and a second that meets the minimum level. The third shows a publication with unknown accessibility.</p>
 
-					<p>The "<a href="#additional-accessibility-information">Additional accessibility information</a>"
-						section is first shown as descriptive statements followed by an example that shows a single
+					<p>The detailed conformance information section is first shown as descriptive statements followed by an example that shows a single
 						compact statement that covers the whole conformance group. Either the descriptive or compact
 						style is appropriate, but do not provide both approaches.</p>
 
-					<aside class="example" title="Additional Accessibility Information with descriptive statements">
-						<p>This publication meest accepted accessibility standards.</p>
+					<aside class="example" title=" descriptive statements with detailed conformance information">
```

Comment:
It's also unclear what "descriptive statements" refers to here. From the old title, it sounds like it was describing the text of the Additional Accessibility Information section. But now it seems to be describing the two paragraphs before it. But when I read the next example, it says it has compact statement yet the paragraphs before the section are exactly the same. I think we're trying to make some distinction about the way the conformance information is written, but having only the diffs to review I'm not sure what it is. I think the titles need clarification, though.